frontend/src/metabase/query_builder/actions/core/initializeQB.unit.spec.ts

```diff
@@ -832,5 +832,10 @@ describe("QB Actions > initializeQB", () => {
       });
       expect(result.uiControls.isShowingNewbModal).toBeFalsy();
     });
+
+    it("initializes `isNativePreviewSidebarOpen` value as false when it is undefined", async () => {
+      const { result } = await setupOrdersTable();
+      expect(result.uiControls.isNativePreviewSidebarOpen).toBe(false);
+    });
```

Comment:
This test is just recycling the existing setup. There is no other reason for it to live under this specific `describe` block. This setting does not apply to any table, question or a model specifically. It is "global".

LMK if there's a better place for this test.